### PR TITLE
Update/readme for launch

### PR DESCRIPTION
**Pull request by wojtekn**
Related to https://github.com/Automattic/dotcom-forge/issues/5765

## Proposed Changes

I propose to update the readme based on the @alexapeduzzi proposal.

## Testing Instructions

Review readme: https://github.com/Automattic/studio/blob/update/readme-for-launch/README.md

## Pre-merge Checklist

- [x] Have you checked for TypeScript, React or other console errors?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,32 @@
 
 A desktop application for creating local WordPress environments powered by WordPress Playground.
```
Comment (alexapeduzzi) [inline README.md]: Suggested swap: take it or leave it! Basically just adding WordPress.com into the mix as well. We have both on our landing page.

"A desktop application for creating local WordPress environments, powered by WordPress.com and WordPress Playground."

File: README.md [hunk 2]
```diff
@@ -2,9 +2,32 @@
...
+**Step 3: Give Feedback**
+
+We’d love to get your thoughts and feedback on your experience with Studio. If you have a question or suggestion, [reach out to our Happiness Engineers](https://developer.wordpress.com/contact/). Also, see the Contributing section for more information on how to get in touch with us.
+
+## Contributing
+
+Studio is an open source project that welcomes all contributions. If you spot a bug or the feature you need is missing, open an issue or propose a Pull Request to implement it.
+
+- Reporting bugs: Open an issue in the repository.
+- Ideas or feature Requests: open an issue in the repository.
```
Comment (alexapeduzzi) [inline README.md]: Capitalize the "O" in Open.

File: README.md [hunk 3]
```diff
@@ -2,9 +2,32 @@
...
+- Code contributions: See the sections below.
+
+### Start development
+
+#### Initial setup
 
 The project includes native dependencies which require Python and it's `setuptools` module to build correctly.
```
Comment (alexapeduzzi) [inline README.md]: Change "it's" to "its"

File: README.md [hunk 4]
```diff
@@ -2,9 +2,32 @@
...
 YMMV but if you manage packages with Homebrew you can do the following:
```
Comment (alexapeduzzi) [inline README.md]: YMMV? Maybe spell this out if it's an acronym.

**Review by alexapeduzzi — CHANGES_REQUESTED** [review]

**Review by sejas — APPROVED** [review]
Thanks for the changes! 👍 
Looks good to me.

File: README.md [hunk 5]
```diff
@@ -1,19 +1,45 @@
...
+**Step 3: Give Feedback**
+
+We’d love to get your thoughts and feedback on your experience with Studio. If you have a question or suggestion, [reach out to our Happiness Engineers](https://developer.wordpress.com/contact/). Also, see the Contributing section for more information on how to get in touch with us.
+
+## Contributing
+
+Studio is an open source project that welcomes all contributions. If you spot a bug or the feature you need is missing, open an issue or propose a Pull Request to implement it.
+
+- Reporting bugs: Open an issue in the repository.
+- Ideas or feature Requests: Open an issue in the repository.
```
Comment (alexapeduzzi) [inline README.md]: Lowercase "R" in "requests"

File: README.md [hunk 6]
```diff
@@ -27,9 +53,11 @@ The app automatically launches with the Chromium developer tools opened by defau
 
 `src/renderer.ts` is the entry point for the "renderer"—the code running in the Chromium window.
 
-Code formatting has been set up to make merging PRs easier. It uses the same prettier/eslint mechanism as Calypso, see p4TIVU-9Lo-p2 for details on setting up your editor.
+#### Code formatting
+
+Code formatting has been set up to make merging PRs easier. It uses the same prettier/eslint mechanism as Calypso, see [JavaScript Coding Guidelines](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/javascript.md) for details on setting up your editor.
```
Comment (alexapeduzzi) [inline README.md]: Change the `,` after Calypso to a period. Also capitalize "See."

**Review by alexapeduzzi — COMMENTED** [review]
Good to go other than these small tweaks!

**Review by alexapeduzzi — APPROVED** [review]

**Pull request merged**